xrviz/__init__.py

```diff
@@ -3,3 +3,11 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+
+def example():
+    import xarray as xr
+    from . import dashboard
+    ds = sample_data.great_lakes
```

Comment:
It would be great if `sample_data` were imported here for the first time, rather than on line 1, so that it doesn't get loaded in sessions that don't use it.

Reply:
A user may want to do have a look at data, by doing `data = xrviz.smaple_data` from anywhere else so I think it is needed.

Reply:
I disagree, I think such a user should explicitly do `import xrviz.sample_data` (which, of course, we document)

Reply:
Corrected. Thanks!